src/util/log.cc

```diff
@@ -542,28 +542,20 @@ std::ostream& operator<<(std::ostream& out, const std::exception_ptr& eptr) {
         // Print more information on some familiar exception types
         try {
             throw;
-        } catch (const seastar::nested_exception& ne) {
```

Comment:
I'm sorry, you cannot remove this without explaining why or what's the implication. This seastar::nested_exception was added by somebody deliberately, and to remove it you need to explain why it's no longer needed - e.g., maybe (?) seastar::nested_exception is already an std::exception has has a good what()?

Reply:
I think it removed it my mistake. The idea was to preserve to current verbosity.
Thanks for catching.